07.ls_object/lib/display.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require_relative 'file_status'
+
+class Display
+  COLUMN_COUNT = 3
+  SHORT_LIST_PADDING = 7
+  FILETYPES = {
+    'fifo' => 'p',
+    'characterSpecial' => 'c',
+    'directory' => 'd',
+    'blockSpecial' => 'b',
+    'file' => '_',
+    'link' => 'l',
+    'socket' => 's'
+  }.freeze
+  MODE_TABLE = {
+    '0' => '---',
+    '1' => '--x',
+    '2' => '-w-',
+    '3' => '-wx',
+    '4' => 'r--',
+    '5' => 'r-x',
+    '6' => 'rw-',
+    '7' => 'rwx'
+  }.freeze
+
+  def initialize(paths)
+    @file_statuses = paths.map { |path| FileStatus.new(path) }
+  end
+
+  def render(long: false)
+    if long
+      render_long_list
+    else
+      width = max_base_name_width + SHORT_LIST_PADDING
+      formated_file_statuses = @file_statuses.map { |file| file.base_name.ljust(width) }
+      file_count = @file_statuses.count
+      row_count = (file_count / COLUMN_COUNT.to_f).ceil
+      (COLUMN_COUNT - (file_count % COLUMN_COUNT)).times { formated_file_statuses << '' } if file_count % COLUMN_COUNT != 0
+      render_lines(formated_file_statuses, row_count)
```

Comment:
```suggestion
      render_hogehoge
```

`render_long_list` と対称性を持たせた方がきれいかなと思います

Reply:
後半の処理を`render_short_list`にまとめることで、バランスが悪い状態だったのをなるべく対称的になるように修正しました。